punjab/httpb.py

```diff
@@ -82,12 +83,12 @@ def _onStartElement(self, name, attrs):
             defaultUri = self.defaultNsStack[1]
 
         if defaultUri and currentUri != defaultUri:
-            raw_xml = u"""<%s xmlns='%s'%s""" % (qname[1], qname[0], '%s')
+            raw_xml = """<%s xmlns='%s'%s""" % (qname[1], qname[0], '%s')
         else:
-            raw_xml = u"""<%s%s""" % (qname[1], '%s')
+            raw_xml = """<%s%s""" % (qname[1], '%s')
 
         # Process attributes
-        for k, v in attrs.items():
+        for k, v in list(attrs.items()):
```

Comment:
remove `list`?

Reply:
No. We need to iterate over a copy attrs is changed in the loop.